OpenRA.Mods.Common/ServerTraits/MasterServerPinger.cs

```diff
@@ -64,7 +65,7 @@ public void Tick(S server)
 
 		public void ServerStarted(S server)
 		{
-			if (!server.Ip.Equals(IPAddress.Loopback) && LanGameBeacon != null)
+			if (!server.Address.Host.Equals(IPAddress.Loopback.ToString()) && LanGameBeacon != null)
```

Comment:
This will obviously fail if the IPv4 loopback address isn't reachable. Using the hostname `localhost` should work for all cases.